app/javascript/gobierto_data/webapp/components/landingviz/VisualizationsGrid.vue

```diff
@@ -13,18 +13,16 @@
           <template v-slot:title>
             {{ name }}
           </template>
-          <Visualizations
-            v-if="items"
-            :items="items"
-            :object-columns="columns"
-            :config="config"
-          />
+          <img
```

Comment:
Would it be possible to fallback and if config.base64 is null render the Visualization component?

Reply:
Yes, done.